test/powershell/engine/Basic/DefaultCommands.Tests.ps1

```diff
@@ -407,6 +407,7 @@ Describe "Verify approved aliases list" -Tags "CI" {
 "Cmdlet", "Set-PSDebug",                                        ,                     $($FullCLR -or $CoreWindows -or $CoreUnix)
 "Cmdlet", "Set-PSSessionConfiguration",                         ,                     $($FullCLR -or $CoreWindows -or $CoreUnix)
 "Cmdlet", "Set-Service",                                        ,                     $($FullCLR -or $CoreWindows              )
+"Cmdlet", "Remove-Service",                                     ,                     $($FullCLR -or $CoreWindows              )

```

Comment:
Can you move this up in the list so it is in alphabetical order? Should go between `Remove-PSSnapin` and `Remove-TypeData`